mkdocs.yml

```diff
@@ -7,13 +7,30 @@ repo_url: https://github.com/aquasecurity/trivy
 edit_uri: ""
 
 nav:
-  - HOME: index.md
-  - Getting started:
-      - Overview: getting-started/overview.md
-      - Installation: getting-started/installation.md
-      - Quick Start: getting-started/quickstart.md
-      - Further Reading: getting-started/further.md
-  - Docs:
+  - Getting Started: 
+      - Overview: index.md
+      - Quick Start: home/quickstart.md
+      - Installation: home/installation.md
+  - Tutorials:
+      - Overview: tutorials/overview.md
+      - Trivy Kubernetes: 
+          - Trivy K8s: tutorials/kubernetes/trivy-kubernetes.md
+          - OVHcloud: tutorials/kubernetes/ovhcloud.md
+      - CI/CD:
+          - Overview: tutorials/integrations/index.md
+          - GitHub Actions: tutorials/integrations/github-actions.md
+          - CircleCI: tutorials/integrations/circleci.md
+          - Travis CI: tutorials/integrations/travis-ci.md
+          - GitLab CI: tutorials/integrations/gitlab-ci.md
+          - Bitbucket Pipelines: tutorials/integrations/bitbucket.md
+          - AWS CodePipeline: tutorials/integrations/aws-codepipeline.md
+          - AWS Security Hub: tutorials/integrations/aws-security-hub.md
+          - GitOps: tutorials/integrations/gitops.md       
+      - Additional Resources: 
+          - Tools: tutorials/other-resources/tools.md
+          - References: tutorials/other-resources/references.md
+          - CKS Reference: tutorials/other-resources/cks.md
```

Comment:
I cannot comment on the file so commenting here: we could add some context there about what is CKS and why it's in the trivy docs.

Reply:
Yes, will add that